docs/content/ecs/accessing-worker-info.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
We have two headings in a row - we can probably drop the top one and promote the others